packages/@aws-cdk-testing/framework-integ/test/aws-eks/test/integ.eks-cluster.ts

```diff
@@ -43,7 +43,7 @@ class EksClusterStack extends Stack {
       vpcSubnets,
       mastersRole,
       defaultCapacity: 2,
-      ...getClusterVersionConfig(this),
+      ...getClusterVersionConfig(this, eks.KubernetesVersion.V1_32),
```

Comment:
I don't think it makes sense to test V1_24 by default.
I plan to fix it in another pull request.
https://github.com/aws/aws-cdk/blob/58c2631de585b300cf8573ab423dcc75791cc3d2/packages/%40aws-cdk-testing/framework-integ/test/aws-eks/test/integ-tests-kubernetes-version.ts#L22-L23